recipes/hydra-submitit-launcher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hydra-submitit-launcher" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e14c8eb46d020fac60ba25f82bcc368dc55851d2683dc95c88631ffcf15e4a34
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - wheel
+    - read-version
+    - pip
+  run:
+    - python >=3.6
+    - hydra-core >=1.1.0.dev7
+    - submitit >=1.3.3
+
+test:
+  imports:
+    - hydra_plugins.hydra_submitit_launcher
+    - hydra
+    - submitit
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/facebookincubator/submitit
```

Comment:
Can you ask upstream if that is the correct home for this project? I would've guess https://github.com/facebookresearch/hydra is instead.